### PR TITLE
752 miovision standardize definition of day

### LINKED ISSUE #752: Miovision daily table should use a more standard definition of "day"
The new `miovision_api.volumes_daily` table is really exciting! This could save me hours in updating the volumes index on a semi-regular basis and make large scale data QA a lot quicker and easier. 

However I notice that the definition of what a "day" is in the daily table is from 11pm of the prior date to 11pm of the current date. This seems very non-standard and likely to cause confusion. 

I think the default assumption would be that the "day" should start and end at midnight. However I think an argument could also be made that a functional day starts and ends closer to 1-2AM when most traffic volumes drop to their minimum. This would allow e.g. weekend nightlife type activities to be lumped into a more meaningful unit. So a late "friday night" could extend to 2am, etc. 

I could go either way on that. What would the rationale be for breaking it at 11pm though? Volumes would still be declining and the date hasn't even changed.

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
- Standardizes definition of a day. No more 11pm-11pm aggregations. 
  - `volumes_15min` and `volumes_15min_mvt` were straightforward to update, just removing `- interval '1 hour'` references. Most of the changed lines come from fluffing these old scripts. 
  - `volumes_daily` was able to be simplified after changing the time period, removing two now unnecessary columns. 

## Issue(s) this solves:

- Closes #752 

## What, in particular, needs to reviewed:
- 

## What needs to be done by a sysadmin after this PR is merged
- [x] Update definitions of four modified functions in bigdata.

- Once this PR is merged and we switch from 11pm-11pm aggregation to 12am-12am aggregation, there will be 1 hour that needs to be manually aggregated for `volumes_15min` and `volumes_15min_mvt`:  
- [x] Update `volumes_15min_mvt`
```sql
    WITH aggregate_delete AS (
        --delete existing aggregation.
        DELETE FROM miovision_api.volumes_15min_mvt
        WHERE
            datetime_bin >= start_date
            AND datetime_bin < end_date
        RETURNING volume_15min_mvt_uid
    )

    --To update foreign key for 1min bin table
    UPDATE miovision_api.volumes AS a
    SET volume_15min_mvt_uid = NULL
    FROM aggregate_delete AS b
    WHERE
        a.volume_15min_mvt_uid = b.volume_15min_mvt_uid
        AND a.datetime_bin >= start_date
        AND a.datetime_bin < end_date;

--insert new aggregation
SELECT miovision_api.aggregate_15_min_mvt(start_date, end_date);
```
- [x] Update `volumes_15min`
```sql
    --delete existing records
    DELETE FROM miovision_api.volumes_15min
    WHERE
        datetime_bin >= start_date
        AND datetime_bin < end_date; 

    --remove processed flag in volumes_15min_mvt
    UPDATE miovision_api.volumes_15min_mvt
    SET processed = NULL
    WHERE 
        datetime_bin >= start_date
        AND datetime_bin < end_date;

--insert new aggregation 
SELECT miovision_api.aggregate_15_min(start_date, end_date);
```
- And for `volumes_daily` we need to drop these two columns, replace function and replace the table contents: 
- [x] Update `volumes_daily` table: 
```sql
ALTER TABLE miovision_api.volumes_daily DROP COLUMN period_start;
ALTER TABLE miovision_api.volumes_daily DROP COLUMN period_end;
--volumes/miovision/sql/function-aggregate-volumes_daily.sql
CREATE OR REPLACE FUNCTION miovision_api.aggregate_volumes_daily....
TRUNCATE TABLE miovision_api.volumes_daily;
SELECT miovision_api.aggregate_volumes_daily('2019-01-01'::date, date_trunc('day', now())::date)
```

### CODE REVIEW

**Comment by chmnata** [comment]
Thanks @gabrielwol! Nice floff. While we are at it can we also change the table alias name from `a` `b` to something more meaningful?

**Comment by gabrielwol** [comment]
@chmnata thanks for the review! I updated the most egregious function: https://github.com/CityofToronto/bdit_data-sources/pull/763/commits/b40399faf8d3703f1daf2866a8d9e08f828c9190

File: volumes/miovision/sql/function-aggregate-volumes_15min.sql [hunk 1]
```diff
@@ -9,49 +9,78 @@ AS $BODY$
...
+            AND atr.classification_uid = ids.classification_uid
+            AND atr.leg = ids.leg
+            AND atr.dir = ids.dir
         ORDER BY volume_15min_uid
         RETURNING volume_15min_mvt_uid
     )
+
     --Sets processed column to TRUE
-    UPDATE miovision_api.volumes_15min_mvt a
+    UPDATE miovision_api.volumes_15min_mvt AS a
```
Comment (chmnata) [inline volumes/miovision/sql/function-aggregate-volumes_15min.sql:77]: this one too plzz

Comment (gabrielwol) [reply volumes/miovision/sql/function-aggregate-volumes_15min.sql]: Done!

**Review by chmnata — APPROVED** [review]

**Pull request merged**